### PR TITLE
GraphQL: Remove viewer from logout mutation

### PR DESCRIPTION
**Pull request by Moumouls**
I also updated mongo db runner to direct github master branch repo since 4.8.0 (published in Jan 2020) do not work on Node 14/15. We need to wait for their new package release. (https://github.com/mongodb-js/runner)

### CODE REVIEW

**Comment by davimacedo** [comment]
Thanks for the PR. It looks good to me. But Travis is not working now and we will have to wait it to check the tests. About the mongodb-runner @dplewis  do you have any thoughts?

**Comment by Moumouls** [comment]
yeah i see that travis was not launched, and also what do you think about switching asap from Travis to Github action ?

**Comment by davimacedo** [comment]
That's an idea. Do you have experience with GitHub actions?

**Comment by Moumouls** [comment]
> That's an idea. Do you have experience with GitHub actions?

Yes, i currently use GitHub action for many projects :), I think the migration could be easy, I can give a try asap !

**Comment by davimacedo** [comment]
I am not it right now. I will push what I've done so far so you can double check.

**Comment by davimacedo** [comment]
https://github.com/parse-community/parse-server/tree/githubActions

**Comment by mtrezza** [comment]
Note: Labels removed to comply with https://github.com/parse-community/parse-server/issues/6984. Labels apply to issues only.

**Comment by Moumouls** [comment]
sorry @mtrezza  i should check the new labeling policy

**Comment by mtrezza** [comment]
@Moumouls No worries, it's still experimental and we're figuring things out as we go. Instructions on label usage will be improved to be easier readable once we have figured this out. If you have any suggestions please feel free to post in the referenced thread 🙂

**Comment by Moumouls** [comment]
@davimacedo LGTY ? 😄

File: package.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
     "jsdoc": "3.6.3",
     "jsdoc-babel": "0.5.0",
     "lint-staged": "10.2.3",
-    "mongodb-runner": "4.8.0",
+    "mongodb-runner": "mongodb-js/runner",
```
Comment (davimacedo) [inline package.json:90]: I just wanted to hear from @dplewis about this change. Otherwise we are good to go.

**Review by davimacedo — APPROVED** [review]

**Pull request merged**